mos-platform/cpm65/cpm.h

```diff
@@ -95,36 +95,74 @@ extern uint8_t cpm_errno;
 /*  6 */ extern uint8_t cpm_conio(uint8_t b);
 /*  7 */ extern uint8_t cpm_get_iobyte(void);
 /*  8 */ extern void cpm_set_iobyte(uint8_t iob);
-/*  9 */ extern void cpm_printstring(const char* s); /* $-terminated */
+/*  9 */ extern void cpm_printstring_i(uint16_t s);
+         static void cpm_printstring(const char* s) /* $-terminated */
+            { cpm_printstring_i((uint16_t) s); }
 /* 10 */ extern uint8_t cpm_readline(uint8_t* buffer);
 /* 11 */ extern uint8_t cpm_const(void);
 /* 12 */ extern uint16_t cpm_get_version(void);
 /* 13 */ extern void cpm_reset_disk_system(void);
 /* 14 */ extern void cpm_select_drive(uint8_t disk);
-/* 15 */ extern uint8_t cpm_open_file(FCB* fcb);
-/* 16 */ extern uint8_t cpm_close_file(FCB* fcb);
-/* 17 */ extern uint8_t cpm_findfirst(FCB* fcb);
-/* 18 */ extern uint8_t cpm_findnext(FCB* fcb);
-/* 19 */ extern uint8_t cpm_delete_file(FCB* fcb);
-/* 20 */ extern uint8_t cpm_read_sequential(FCB* fcb);
-/* 21 */ extern uint8_t cpm_write_sequential(FCB* fcb);
-/* 22 */ extern uint8_t cpm_make_file(FCB* fcb);
-/* 23 */ extern uint8_t cpm_rename_file(RCB* rcb);
+/* 15 */ extern uint8_t cpm_open_file_i(uint16_t fcb);
+         static uint8_t cpm_open_file(FCB* fcb)
```

Comment:
Nice, these should just get inlined away!

There's a sliiight smell here, in that cpm_open_file in one C file won't == cpm_open_file in another. Since we use LTO, you could just move these wrappers to a C file, rather than a header. The compiler should still inline them all the same, since it has a global view of all calls and definitions.

Reply:
Done.